charts/pgbouncer/templates/_helpers.yaml

```diff
@@ -0,0 +1,70 @@
+{{- define "pgbouncer.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "pgbouncer.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+
+{{ define "pgbouncer_image" -}}
+{{ printf "%s:%s" .Values.image.repository .Values.image.tag }}
+{{- end }}
```

Comment:
This need to be refactored to support private registry. EG: https://github.com/astronomer/astronomer/blob/276a732ee3/charts/astronomer/templates/_helpers.yaml#L36-L42

Reply:
good call, will do.

Reply:
fixed.